### PR TITLE
Add CSS variables on modal component

### PR DESCRIPTION
**Pull request by branquinhoaa**
If applied, this pull request will allow consumers to customize the style for modal.

### Other minor changes:
N/A

### Card Link:
N/A

### Design Expected Screenshot
N/A

### Implementation Screenshot or GIF
Screenshot from 2021-12-16 14-47-30

### Notes:
N/A

### CODE REVIEW

File: src/components/Modal/Modal.css [hunk 1]
```diff
@@ -1,28 +1,23 @@
...
-  --modal-container-background-color: rgba(0, 0, 0, 0.1);
-}
 
 .container {
-  display: none;
-  background-color: var(--modal-container-background-color);
-  display: flex;
-  justify-content: center;
-  align-items: center;
+  display: var(--modal-container-display, flex);
```
Comment (kaiomagalhaes) [inline src/components/Modal/Modal.css:5]: you're moving from none to flex by default, is there a reason for that?

File: src/components/Modal/Modal.css [hunk 2]
```diff
@@ -1,28 +1,23 @@
...
+  margin: var(--modal-container-align-items, auto);
 }
 
 .content-container {
-  background-color: white;
-  border: var(--modal-border);
-  border-radius: 5px;
-  height: auto;
-  width: auto;
+  background-color: var(--modal-content-container-background-color,white);
```
Comment (kaiomagalhaes) [inline src/components/Modal/Modal.css:18]: can you please use css color variables here?

**Review by kaiomagalhaes — CHANGES_REQUESTED** [review]

File: src/components/Modal/Modal.css (hunk 1 above)
Comment (paulohfev) [reply src/components/Modal/Modal.css:4]: hey @branquinhoaa can you please clarify this doubt?

File: src/components/Modal/Modal.stories.mdx [hunk 3]
```diff
@@ -37,7 +37,15 @@ export default SimpleModal;
...
 ## CSS Variables available
 
-|  Property   |            Attribute             |         State        |
-| :---------: | :------------------------------: | :------------------: |
-|             |          modal-border            |                      |
-|             | modal-container-background-color |                      |
\ No newline at end of file
+|     Property      |    Attribute     | State |
+| :---------------: | :--------------: | :---: |
+|     container     |     display      |       |
```
Comment (paulohfev) [inline src/components/Modal/Modal.stories.mdx:42]: @branquinhoaa please order the attributes by alphabetical order

**Review by paulohfev — CHANGES_REQUESTED** [review]

**Pull request merged**